tools/appliance/systemvmtemplate/template.json

```diff
@@ -32,7 +32,7 @@
         [ "-m", "512M" ],
         [ "-smp", "cpus=1,maxcpus=1,cores=1" ]
       ],
-      "disk_size": 1800,
+      "disk_size": 2100,
```

Comment:
We spent a lot of time optimizing for the size, I only see an increase in 200MB, should be fixed at 2000 then.